### DISCUSSION TITLE
Player Rater, ADP, and add/drop stats

### DISCUSSION THREAD

**Discussion by equilibliam** [cwendt94/espn-api#307] [ESPN API]
Hello, 

Is there a way to pull the player rater stats and/or the ADP trends and Add/Drop trends from player data? I do not see it right now but I could be missing it. 

The player rater is pretty bad at getting real player values on ESPN but I would love to be able to see the difference between my own value calcs and ESPNs to add another layer to my trade suggestion tool. ADP trends and Add/Drop numbers would also be helpful to have around to check some more boxes and do analysis for upcoming drafts.

Thanks for building this API - it has been tremendously helpful in making my team a powerhouse in fantasy basketball this year.

**Comment by dtcarls** [comment]
Here is a gist to grab football auction ADP. I'm sure it could be tweaked to get the data you need or integrate it into the package. 

https://gist.github.com/dtcarls/c6158cd8ad41e0941571807052098a0f
